pandas/conftest.py

```diff
@@ -65,3 +67,22 @@ def ip():
     pytest.importorskip('IPython', minversion="6.0.0")
     from IPython.core.interactiveshell import InteractiveShell
     return InteractiveShell()
+
+
+@pytest.fixture(params=[tm.makeUnicodeIndex(100),
+                        tm.makeStringIndex(100),
+                        tm.makeDateIndex(100),
+                        tm.makePeriodIndex(100),
+                        tm.makeTimedeltaIndex(100),
+                        tm.makeIntIndex(100),
```

Comment:
this should work but i was actually meaning a conftest 
in the tests/indexes/conftest.py

these put the fixtures close to the testing code

Reply:
In case this comes up for anyone else, I got confused about the scope of conftest.py until reading this post: https://stackoverflow.com/questions/34466027/in-py-test-what-is-the-use-of-conftest-py-files